packages/cli/src/active-workflow-manager.ts

```diff
@@ -233,14 +228,10 @@ export class ActiveWorkflowManager {
 	 * deregister those webhooks from external services.
 	 */
 	async clearWebhooks(workflowId: WorkflowId) {
-		const workflowData = await this.workflowRepository.findOne({
+		const workflowData = await this.workflowRepository.findOneOrFail({
```

Comment:
I assume it's intentional that this now becomes an error that will be reported to Sentry?

Reply:
Yes. It should already be reported, as the default level for `ApplicationError` is `error`. The error type does change now to `EntityNotFoundError`